src/index.spec.js

```diff
@@ -332,57 +336,59 @@ describe('Dropzone', () => {
 
     it('should set proper dragActive state on dragEnter', () => {
       const dropzone = mount(
-        <Dropzone />
+        <Dropzone {...classConfig} />
       );
       dropzone.simulate('dragEnter', { dataTransfer: { files } });
-      expect(dropzone.state().isDragActive).toEqual(true);
-      expect(dropzone.state().isDragReject).toEqual(false);
+      expect(dropzone.hasClass(classConfig.activeClassName)).toEqual(true);
```

Comment:
I'll deprecate classNames soon and this will cause troubles in the future. Could we check props on children?